### PR TITLE
Task5

### PR DESCRIPTION
**Pull request by YuruMaksau**
I fixed some comments.

### CODE REVIEW

**Comment by pahuta** [comment]
What is the folder "Task4" doing here? Just remove it from the branch "Task5"

File: Task5/index.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Task5</title>
+   <link rel="stylesheet" href="css/style.css">
```
Comment (pahuta) [inline Task5/index.html]: Formatting issue

File: Task5/css/button.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    padding: 0;
+    border-radius: 5px;
+    color: #ffffff;
+    font-weight: bold;
+    margin: 0 5px 8px 0;
+}
+
+.button_small {
+    height: 20px;
+    width: 75px;
```
Comment (pahuta) [inline Task5/css/button.css]: Specifying `width` and `height` it's a wrong way. Try to change caption  of  "small" button to "My super-puper mega multi button" and see the result. Right way is a specifying a padding.

File: Task5/css/button.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.layout {
+    margin-top: 10px;
+    padding: 15px;
+    border: 1px solid #beb0b0;
+}
```
Comment (pahuta) [inline Task5/css/button.css:5]: Does the class `.layout` relate to the buttons? Why it specified in the `button.css` file?

File: Task5/css/header.css [hunk 4]
```diff
@@ -0,0 +1,114 @@
+body {
+    width: 1200px;
+    margin: 0 auto;
+}
+
+header {
+    height: 250px;
+    width: 1200px;
```
Comment (pahuta) [inline Task5/css/header.css]: `width: 1200px;` - a redundant property here

File: Task5/css/header.css [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+    position: relative;
+    z-index: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.signboard {
+    width: 370px;
+    height: 100px;
```
Comment (pahuta) [inline Task5/css/header.css]: The same as for buttons: use padding instead width/height. Try add additional text to to the "signboard". It's not change its width dynamically.

File: Task5/css/header.css [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    width: 360px;
+    height: 90px;
+}
+
+.inner {
+    width: 350px;
+    height: 80px;
+    border: 2.5px solid;
+    padding: 0;
+}
```
Comment (pahuta) [inline Task5/css/header.css]: Where do you use the class `.inner`?

File: Task5/css/header.css [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+.icon {
+    display: inline-block;
+    background: url("../image/spice-social-gadget-sprite.png") no-repeat;
+    background-size: 1100% 300%;
+    width: 45px;
+    height: 45px;
+    margin-bottom: 20px;
+    margin-right: 20px;
```
Comment (pahuta) [inline Task5/css/header.css]: Instead two properties `margin-*` you can specify one `margin: 10px;` and get the same result.

File: Task5/css/header.css [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+.social {
+    transform: rotate(-1.7grad);
+    height: 300px;
+    width: 1350px;
+}
+
+.icon {
+    display: inline-block;
```
Comment (pahuta) [inline Task5/css/header.css]: `display: inline-block;` - a redundant property. Isn't it?

File: Task5/index.html (hunk 1 above)
Comment (YuruMaksau) [reply Task5/index.html]: fixed

File: Task5/css/button.css (hunk 3 above)
Comment (YuruMaksau) [reply Task5/css/button.css:5]: class .layout removed in the header.css

File: Task5/css/header.css (hunk 6 above)
Comment (YuruMaksau) [reply Task5/css/header.css]: nowhere. class .inner was deleted.

File: Task5/css/header.css (hunk 7 above)
Comment (YuruMaksau) [reply Task5/css/header.css]: fiexed.

File: Task5/css/header.css (hunk 8 above)
Comment (YuruMaksau) [reply Task5/css/header.css]: you right! fixed.

File: Task5/css/header.css (hunk 4 above)
Comment (YuruMaksau) [reply Task5/css/header.css]: you right! fixed!

File: Task5/css/button.css (hunk 2 above)
Comment (YuruMaksau) [reply Task5/css/button.css]: fixed.

**Comment by YuruMaksau** [comment]
All comments were fixed.

File: Task5/css/header.css (hunk 5 above)
Comment (YuruMaksau) [reply Task5/css/header.css]: fixed.

File: Task5/css/header.css [hunk 9]
```diff
@@ -0,0 +1,107 @@
+.layout {
+    margin-top: 10px;
+    padding: 15px;
+    border: 1px solid #beb0b0;
+}
```
Comment (pahuta) [inline Task5/css/header.css]: Does the class `.layout` relate to the header? I see that you don't use it in header. Why it specified in the header.css file?

File: Task5/css/header.css [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+    padding: 36px 64px;
+}
+
+.signboard::before {
+    content: "";
+    position: absolute;
+    z-index: -1;
+    background: #ffffff;
+    border: 3px solid black;
+    padding: 11% 47%;
```
Comment (pahuta) [inline Task5/css/header.css]: `11% 47%` - what mean these magic numbers? Do they have suitable values if we will change a text?

Comment (pahuta) [reply Task5/css/header.css]: Check it https://i.imgur.com/I1nSV1q.png. I guess the board should be at the center of the header block, doesn't matter how many text we have inside it. Fix it! Also check the white "border" outside from the black one.

File: Task5/css/header.css (hunk 9 above)
Comment (YuruMaksau) [reply Task5/css/header.css]: At that moment I endured class .layout in the button! In my opinion .layout must situated in a separate file css.

File: Task5/css/header.css (hunk 10 above)
Comment (YuruMaksau) [reply Task5/css/header.css]: Comments was fixed.

File: Task5/css/header.css (hunk 9 above)
Comment (pahuta) [reply Task5/css/header.css]: So, Why didn't you create a new file for it? Fix it in next tack.

**Pull request merged**